tests/main.c

```diff
@@ -3,6 +3,10 @@
 
 #include <stdio.h>
 
+#ifdef USE_VLD
+#include "vld.h"
+#endif
```

Comment:
I expect a major copy/pasta coming.
We should probably change our ps1 that checks for vld. vld_check.bat in the main repo will up to now fail builds if vld.h is detected to be included.